src/task2/repository/usersRepository.js

```diff
@@ -0,0 +1,20 @@
+const dataBase = new Map();
```

Comment:
ideally it should be implemented using such tools like https://www.npmjs.com/package/lowdb
but, we can keep it, because next task will be be to connect with real DB

Reply:
Thank you for the information, I've looked for something like that, but unfortunately didn't found